DirectProgramming/DPC++FPGA/include/MemoryTransfers.hpp

```diff
@@ -0,0 +1,296 @@
+#pragma once
```

Comment:
All file names should be `snake_case`, even if they contain class the is not `SnakeCase`. Sorry, make sure all files are `snake_case`.